driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -1722,9 +1729,15 @@ private CloseFuture close() {
         logger.debug("Shutting down");
 
         // stop debouncers
-        nodeListRefreshRequestDebouncer.stop();
-        nodeRefreshRequestDebouncer.stop();
-        schemaRefreshRequestDebouncer.stop();
+        if (nodeListRefreshRequestDebouncer != null) {
```

Comment:
Additional null checks to allow for a partially initialized cluster to still be closed

Reply:
It looks overly paranoid, but I think it's worth it :)